varianteval/scripts/README

```diff
@@ -1 +1,62 @@
-Utility scripts
+## Utility scripts
+### annotation_script.py
+Script to add annotation to input callset with respect to specified comparison callsets. Annotations identify
+records in the input callset that overlap with records in the comparison callsets. An example call is of the form:
+```
+python annotation_script.py --output_dir {path} --source_vcf {path} --comparison_vcfs {path1 path2 ...} --comparison_labels {label1 label2 ...}
+```
+The full set of input flags and default values are given here:
+```
+  --output_dir OUTPUT_DIR
+                        output directory for annotated files
+  --source_vcf SOURCE_VCF
+                        input vcf that will have annotations add to (a copy
+                        of) it
+  --comparison_vcfs COMPARISON_VCFS [COMPARISON_VCFS ...]
+                        list of comparison/conflict databases
+  --comparison_labels COMPARISON_LABELS [COMPARISON_LABELS ...]
+                        list of annotation labels corresponding to the input
+                        comparison vcfs
+  --conflict_overlap_thresh CONFLICT_OVERLAP_THRESH
```

Comment:
is the conflict logic still there? -- we should separate it out of the annotations to make it more generalizable and agnostic to what is being compared; suggestion: rename to overlap_thr